RELEASE.md

```diff
@@ -42,3 +42,28 @@ If bug fixes need to be backported to previous releases, a PR through main is al
     - `release-vx.y.1 <- main`
 3. Create a PR to cherry-pick the fix into `release-vx.y.2`.
     - `release-vx.y.2 <- main`
+
+## Security
+
+Tip of the branch from which runtime release is considered should be signed at least by 2 signatures.
+
+These to be owned by Technical committee or by Council members.
```

Comment:
What does it mean to be owned?

Reply:
I think in this sense https://www.babypips.com/crypto/learn/what-is-a-digital-signature   .  The only owner of private key can sign by key. So he kind of owns ability to sign. It is shortcut to avoid explaining that you own the key. And if you sign something then it is unique. owner => key => signature === owner => signature

Reply:
Eight the breakup of the paragraph threw me off lol

Reply:
```suggestion
```